lms/envs/common.py

```diff
@@ -1928,6 +1928,11 @@
 SHIBBOLETH_DOMAIN_PREFIX = 'shib:'
 OPENID_DOMAIN_PREFIX = 'openid:'
 
+### SHIB
+# For SHIB backup register and login URLs
+SHIB_ONLY_SITE = False
+SHIB_REDIRECT_DOMAIN_WHITELIST = {}
```

Comment:
maybe you want to document how to set this setting (the dict with lists as values?)